index.js

```diff
@@ -13,6 +13,7 @@
 /** @typedef {import('stream').TransformOptions} TransformOptions */
 
 const { Transform } = require('stream');
+const util = require('util');
```

Comment:
@XhmikosR I'm not sure how to proceed here
'node:utils' available after Node v14

Reply:
I suggest you just ignore the rule in package.json and we re-enable it after svg-sprite v3.0.0 where we drop support for Node.js < 14